docs/vagrant.md

```diff
@@ -22,6 +22,9 @@ Running with Vagrant
     * `splunkforwarder-6.2.5-272645-linux-2.6-x86_64.rpm` and `splunkforwarder-6.2.5-272645-linux-2.6-amd64.deb` ~ 13MB each
 * `vagrant-omnibus` installer currently requires internet access to function.
 
+<b>Note</b>: After you spin up the cluster slaves you will have to restart the cluster master to bring the cluster to a stable state. While spinning up the cluster slaves, they are re-assigned with different GUIDS which requires a restart(this restart can take a while to complete). Cluster master is restarted so that it can identify the new GUIDS.
```

Comment:
Just a note for styling, you can get bold text in markdown without HTML (which I consider a last resort) with two asterisks or two underscores (i.e. `**bold**` or `__bold__`).